test/config.json

```diff
@@ -422,6 +422,18 @@
 			"RetryMax": 0,
 			"Tags": []
 		},
+		"cluster_endtoend": {
+			"File": "",
+			"Args": [],
+			"Command": [
+				"make",
+				"e2e_test_cluster"
+			],
+			"Manual": false,
+			"Shard": 2,
+			"RetryMax": 0,
+			"Tags": []
+		},
```

Comment:
I have added a GitHub action for this, so we I think we should move this to Shard 5 (do not execute in Travis).

Reply:
Changes done!